### DISCUSSION TITLE
GitHub question: 2nd change on a patch, another pull request?

### DISCUSSION THREAD

**Discussion by n4n0b1t3** [smartcontractkit/full-blockchain-solidity-course-py#1113] [Q&A] [answered]
I am such a noob when it comes to github. Currently, I have two patches waiting for a merge. I now added a new section to patch-1. Do I need to do a pull request again, or will an already "pull requested" patch considered for a merge anyway no matter the number of changes I did until the merge?

**Comment by cromewar** [comment, marked as answer]
Depends a lot on the context, however if each one has different changes it should not cause any troubles, however in sake of good organization, I'll suggest to close the oldest ones and send just one pull request with all changes on it.

Cheers.
